extensions/positron-code-cells/src/extension.ts

```diff
@@ -6,18 +6,49 @@
 import * as vscode from 'vscode';
 import { initializeLogging } from './logging';
 import { CellCodeLensProvider } from './codeLenses';
-import { registerCommands } from './commands';
 import { activateDecorations } from './decorations';
 import { activateContextKeys } from './context';
+import { canHaveCells, destroyDocumentManager, getOrCreateDocumentManager } from './documentManager';
+import { registerCommands } from './commands';
 
-export const IGNORED_SCHEMES = ['vscode-notebook-cell', 'vscode-interactive-input'];
+export const IGNORED_SCHEMES = ['vscode-notebook-cell', 'vscode-interactive-input', 'inmemory'];
 
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
 	initializeLogging();
 
+	// When starting extension, fill documentManagers
```

Comment:
What do you think about moving these event listeners to an `activateDocumentManagers` function, similar to the decorations and context key components?